src/nifti-base.ts

```diff
@@ -0,0 +1,636 @@
+import { NiftiExtension } from "./nifti-extension";
+import { Utils, ArrayMatrix3, ArrayMatrix4 } from "./utilities";
+
+export abstract class BaseNIFTI {
```

Comment:
Introduced a base class for the methods that `NIFTI2` used from `NIFTI1`.

Expect this to change a bit as I decide how to deal with the super long constructor params